src/main/java/eu/darkbot/utils/Point.java

```diff
@@ -0,0 +1,15 @@
+package eu.darkbot.utils;
+
+public class Point {
```

Comment:
I'd implement locatable here too, and just transform to double if treated as location